Makefile

```diff
@@ -4,7 +4,8 @@
 
 TARGET = crouton
 EXTTARGET = crouton.zip
-SRCTARGETS = $(patsubst src/%.c,crouton%,$(wildcard src/*.c))
+SRCTARGETS = $(patsubst src/%.c,crouton%,$(shell find src/ -name *.c -and -not -name freon.c))
```

Comment:
Defining LIBS first and using patsubst to provide a pattern to `$(filter-out pattern...,text)` would be way cleaner.